### PR TITLE
El Cantor HS keyboard. 44 key diodeless split keyboard with support for hotswap sockets. Based on diepala/cantor.

### PR DESCRIPTION
**Pull request by azhizhinov**
## Description

The El Cantor HS keyboard is a 44 key diodeless split keyboard with support for hotswap sockets, designed with simplicity in mind. It is based on the popular [cantor](https://github.com/diepala/cantor), which is inspired on the popular [corne](https://github.com/foostan/crkbd), [ferris](https://github.com/pierrechevalier83/ferris) and [sweep](https://github.com/davidphilipbarr/Sweep) keyboards, aiming to provide a more ergonomic (stronger column stagger) corne-like layout with a simple, easy to assemble and cheap design.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by azhizhinov — COMMENTED** [review]
All checks were passed.

**Comment by azhizhinov** [comment]
Thanks for your help.

File: keyboards/elcantorhs/readme.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# El Cantor HS
+
+![El Cantor HS](https://thumb.cloud.mail.ru/weblink/thumb/xw1/VtFZ/jwSN7r6BB)
+
+The El Cantor HS keyboard is a 43 key diodeless split keyboard with support for hotswap sockets, designed with simplicity in mind. It is based on the popular [cantor](https://github.com/diepala/cantor), which is inspired on the popular [corne](https://github.com/foostan/crkbd), [ferris](https://github.com/pierrechevalier83/ferris) and [sweep](https://github.com/davidphilipbarr/Sweep) keyboards, aiming to provide a more ergonomic (stronger column stagger) corne-like layout with a simple, easy to assemble and cheap design.
```
Comment (dunk2k) [inline keyboards/elcantorhs/readme.md:5]: No need to elaborate on Cantor's design influences, this readme.md is all about the El Cantor HS 😉
```suggestion
The El Cantor HS keyboard is a 44 key diodeless split keyboard with support for hotswap sockets, designed with simplicity in mind. Based on the popular [Cantor](https://github.com/diepala/cantor) keyboard, this keyboard features a more aggressive column stagger than the afore mentioned, whilst aiming to simplify assembly and improve cost efficiency.
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Comment by azhizhinov** [comment]
Hello.
Done.

File: keyboards/elcantorhs/readme.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# El Cantor HS
+
+![El Cantor HS](https://thumb.cloud.mail.ru/weblink/thumb/xw1/VtFZ/jwSN7r6BB)
```
Comment (drashna) [inline keyboards/elcantorhs/readme.md:3]: Could you rehost this on imgur or the like?

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by azhizhinov** [comment]
Hello.
Done.

**Comment by azhizhinov** [comment]
Hello.
@drashna
Done. Hosting for the image has been changed.
The image can be removed at all, it's not important in this case at all.
Thanks in advance.

**Comment by azhizhinov** [comment]
Hello.
@drashna 
Is there any reason, why you are not approving this pull request?
Thanks in advance.

**Comment by azhizhinov** [comment]
Hello.
@drashna 
Done. Thanks for your time.

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**